tardis/plasma/properties/plasma_input.py

```diff
@@ -33,8 +35,11 @@ class TRadiative(ArrayInput):
     outputs = ("t_rad",)
     latex_name = (r"T_{\textrm{rad}}",)
 
+    def calculate(self, dilute_planckian_radiation_field):
+        return dilute_planckian_radiation_field.temperature.cgs.value
```

Comment:
Sure we want a value here and not just the quantity?

Reply:
currently plasma needs floats.